zipkin-example/src/main/java/com/example/helloworld/HelloWorldApplication.java

```diff
@@ -19,7 +21,7 @@ public static void main(String[] args) throws Exception {
 
     @Override
     public String getName() {
-        return "hello-world";
+        return "dropwizard-zipkin";
```

Comment:
This was just a sample app so you end up seeing the same service twice anyway, regardless of what we call it (since its just calling itself).  I like the change of not hardcoding the port, but yea, for now, let's revert this name change please.